src/bios.rs

```diff
@@ -142,10 +144,63 @@ impl Component for Bios {
         crate::component::query_adopt_state()
     }
 
+    // Backup the current grub.cfg and install new static config
+    // - Backup "/boot/loader/grub.cfg" to "/boot/grub2/grub.cfg.bak"
+    // - Remove symlink "/boot/grub2/grub.cfg"
+    // - Install new static grub.cfg
+    // - Remove old "/boot/loader/grub.cfg"
+    fn migrate_static_grub_config(&self, sysroot_path: &str, destdir: &openat::Dir) -> Result<()> {
+        use anyhow::Context;
+        use openat_ext::OpenatDirExt;
+
+        // sysroot_path is /, destdir is Dir of /
+        let grub_config_path = Utf8PathBuf::from(sysroot_path).join("boot/grub2");
+        let grub_config_dir = destdir
+            .sub_dir("boot/grub2")
+            .context("Opening boot/grub2")?;
```

Comment:
What the difference between those two paths here?

Reply:
`grub_config_path` is `Utf8PathBuf` of `/boot/grub2`, that will be used to backup.
`grub_config_dir` is `Openat::Dir`, that will be used as `read_link` the real path of `/boot/grub2/grub.cfg` and do sync.
So we need both, WDYT?